test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateDisplayLayers.py

```diff
@@ -70,6 +70,34 @@ def _GetMayaNode(self, nodeName):
         selectionList.add(nodeName)
         return selectionList.getDependNode(0)
 
+    def testDisplayLayersTexturing(self):
+        cmds.file(force=True, new=True)
+        mayaUtils.loadPlugin("mayaUsdPlugin")
+
+        cmds.xform("persp", t=(2, 2, 5.8))
+        cmds.xform("persp", ro=[0, 0, 0], ws=True)
+
+        panel = mayaUtils.activeModelPanel()
+        cmds.modelEditor(panel, edit=True, displayTextures=True)
+
+        testFile = testUtils.getTestScene("multipleMaterialsAssignment",
+                                          "MultipleMaterialsAssignment.usda")
+        proxyDagPath, usdStage = mayaUtils.createProxyFromFile(testFile)
+        cmds.select(cl=True)
+
+        # add the stage to a new display layer
```

Comment:
Technically you are adding the root prim of the stage, not the stage proxy node, but the code reader will probably understand. (Beside, it is only a test.)